lnbits/core/static/js/index.js

```diff
@@ -7,10 +7,61 @@ new Vue({
         show: false,
         data: {}
       },
-      walletName: ''
+      usr: '',
+      username: '',
+      password: '',
+      password_repeat: '',
+      walletName: '',
+      signup: false
     }
   },
   methods: {
+    register: function () {
+      axios({
+        method: 'POST',
+        url: '/api/v1/register',
+        data: {
+          email: this.username,
+          password: this.password,
+          password_repeat: this.password_repeat
+        }
+      })
+        .then(response => {
+          this.$q.cookies.set('access-token', response.data.access_token)
+          this.$q.localStorage.set('lnbits.token', response.data.access_token)
+          this.$q.localStorage.set('lnbits.usr', response.data.usr)
+          window.location.href = '/wallet'
+        })
+        .catch(LNbits.utils.notifyApiError)
+    },
+    login: function () {
+      axios({
+        method: 'POST',
+        url: '/api/v1/login',
+        data: {username: this.username, password: this.password}
+      })
+        .then(response => {
+          this.$q.cookies.set('access-token', response.data.access_token)
+          this.$q.localStorage.set('lnbits.token', response.data.access_token)
+          // this.$q.localStorage.set('lnbits.usr', response.data.usr)
+          window.location.href = '/wallet'
+        })
+        .catch(LNbits.utils.notifyApiError)
+    },
+    login_usr: function () {
```

Comment:
- cammelCase prefered

```suggestion
    loginUser: function () {
```